test/functional/tests/io_class/test_io_class_occupancy_load.py

```diff
@@ -0,0 +1,181 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from collections import namedtuple
+from math import isclose
+
+import pytest
+
+from .io_class_common import *
+from api.cas.cache_config import CacheMode, CacheLineSize
+from api.cas.casadm_params import OutputFormat
+from api.cas.ioclass_config import IoClass
+from storage_devices.device import Device
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools import fs_utils
+from test_tools.disk_utils import Filesystem
+from test_utils.os_utils import sync, Udev
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+def test_ioclass_occuppancy_load(cache_line_size):
+    """
+        title: Load cache with occupancy limit specified
+        description: |
+          Start cache in WB, add ioclasses with a occupancy limit. Populate cache
```

Comment:
I think that the last sentence will be satisfying as a description. Other things like starting cache in WB will be visible in test steps